src/crsAdapter/CetsAdapter.js

```diff
@@ -551,6 +605,48 @@ class CetsAdapter {
         });
     }
 
+    assignHotelServiceFromAdapterObjectToXmlObject(service, xml) {
+        let startDate = moment(service.startDate, this.options.useDateFormat);
+
+        let xmlService = {
+            [CONFIG.builderOptions.attrkey]: {
+                ServiceType: CONFIG.defaults.serviceType.hotel,
+            },
+            Product: service.destination.substring(3),
+            Program: 'HOTEL',
+            Destination: service.destination.substring(0, 3),
+            Room: service.roomCode,
+            Norm: service.roomOccupancy,
+            MaxAdults: service.roomQuantity,
+            Meal: service.mealCode,
+            StartDate: startDate.isValid() ? startDate.format(CONFIG.crs.dateFormat) : service.dateFrom,
+            Duration: this.calculateDuration(service.dateFrom, service.dateTo),
+        };
+
+        xml.Fah.push(xmlService);
+    }
+
+    assignHotelTravellers(service, xml) {
+        if (!service.travellers) return;
+
+        xml.Fap = [];
+
+        service.travellers.forEach((serviceTraveller, index) => {
+            const traveller = this.helper.traveller.normalizeTraveller(serviceTraveller, SERVICE_TYPES.hotel);
+
+            xml.Fap.push({
+                [CONFIG.builderOptions.attrkey]: {
+                    ID: index + 1,
+                },
+                PersonType: traveller.salutation,
+                Name: traveller.firstName + ' ' + traveller.lastName,
+                FirstName: traveller.firstName,
```

Comment:
this is the only place where we need the first name separately. i think it is not correct to increase the complexity of the code only for this. for this case i suggest to keep the old traveller structure and do here only a `traveller.name.split(' ')[0]` to get the first name. if we have to support more CRSs like CETS we need to think about a more general solution.